catalyst_voices/packages/catalyst_voices_models/lib/src/brands.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_assets/generated/colors.gen.dart';
+
+import 'package:flutter/material.dart';
+
+// [brands] is a Map that stores the ThemeData for all the
+// available brands in the Catalyst Voices app.
+// This map is used by the [BrandBloc] to properly
+// switch the active theme based on the BlocState.
+
+final Map<BrandKey, ThemeData> brands = {
+  BrandKey.catalyst: ThemeData(
+    colorScheme: ColorScheme.fromSeed(
+      seedColor: VoicesColors.blue,
+      primary: VoicesColors.blue,
+    ),
+  ),
+  BrandKey.dummy: ThemeData.from(
```

Comment:
Somehow I have a feeling that these brands which are classes very specific to flutter should not live in 'catalyst_voices_models' package/layer since it looks like it was intended to hold "business" entities and be free from flutter framework, especially free of any UI related code. 

Imho these should live where other widgets and UI stuff lives, currently this is catalyst_voices main package.

Blocs rather should be free of any UI stuff too, probably if I were to implement this myself I would:
- keep BrandKey as a state in the BrandBloc, not the ThemeData
- Listen to BrandKey changes where ThemeData is needed (as a parent of MaterialApp) and build the ThemeData using a 'Theme builder' which would live in catalyst_voices, maybe in catalyst_voices_shared but I don't know the purpose of the shared package yet

Reply:
@minikin what's your opinion here?

Reply:
`BrandBloc` now uses only the `BrandKey` as a state.
Rest of the logic and the ThemeBuilder now lives in `catalyst_voices_brands` package.